run-checks

```diff
@@ -267,6 +267,11 @@ if [ "$STATIC" = 1 ]; then
     fi
 
     # FIXME: re-add staticcheck with a matching version for the used go-version
+
+    if [ -z "${SKIP_TESTS_CHECK-}" ] || [ "$SKIP_TESTS_CHECK" = 0 ]; then
+        echo "Checking tests formatting"
+        ./tests/utils/check-test-format -d ./tests
```

Comment:
Let's switch it to positional now. It's a bit awkward if you are required to pass an option with `-d ..`.

```suggestion
        ./tests/utils/check-test-format ./tests
```

Reply:
done